app/src/main/java/com/example/todoapp/viewmodel/CreateTaskViewModel.kt

```diff
@@ -0,0 +1,28 @@
+package com.example.todoapp.viewmodel
+
+import androidx.lifecycle.ViewModel
+import com.example.todoapp.SingleLiveEvent
+import com.example.todoapp.data.local.repository.TaskRepository
+import com.example.todoapp.model.Task
+import io.reactivex.rxjava3.android.schedulers.AndroidSchedulers
+import io.reactivex.rxjava3.schedulers.Schedulers
+
+class CreateTaskViewModel(
+    private val taskRepository: TaskRepository
+): ViewModel() {
+
+    val isCreateSucceed = SingleLiveEvent<Boolean>()
```

Comment:
để tránh các class ngoài CreateTaskViewModel có thể tương tác, và trực tiếp update lại giá trị của isCreateSucceed
em nên viết như sau 
```suggestion
    private val _isCreateSucceed = SingleLiveEvent<Boolean>()
    val isCreateSucceed: LiveData<Boolean> get()= _isCreateSucceed
```